internal/adapters/database/csv_handler.go

```diff
@@ -0,0 +1,51 @@
+package database
+
+import (
+	"encoding/csv"
+	"fmt"
+	"os"
+
+	"github.com/ahadiihsan/brick_scraper_go/internal/entities"
+)
+
+// CSVHandler implements DatabaseHandler
+type CSVHandler struct {
+}
```

Comment:
Too implicit, make it like this to infer more explicitly
```suggestion
// CSVHandler implements DatabaseHandler
var _ DatabaseHandler = (*CSVHandler)(nil)

type CSVHandler struct {
}
```
reference: https://github.com/uber-go/guide/issues/25